### PR TITLE
Feature/ukirw 442 add error messages for running workflow

### PR DESCRIPTION
**Pull request by mlu-ssoft**
Add display errors for running Workflow.

ticket:
- https://ukri-spyrosoft.atlassian.net/browse/UKRIW-442

### CODE REVIEW

File: libs/map/data-access-map/src/lib/mutation/workflow.error.tsx [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+      return 'MAP.ACTION_CREATOR_PANEL.WORKFLOW.NODE.DATA_SET.AUXILIARY.CORINE_LAND_COVER';
+    }
+
+    case 'clms-water-bodies': {
+      return 'MAP.ACTION_CREATOR_PANEL.WORKFLOW.NODE.DATA_SET.AUXILIARY.WATER_BODIES';
+    }
+  }
+
+  return null;
+};
```
Comment (KatarzynaPaszko) [inline libs/map/data-access-map/src/lib/mutation/workflow.error.tsx:94]:
```suggestion
const BASE_KEY = 'MAP.ACTION_CREATOR_PANEL.WORKFLOW.NODE.DATA_SET';

const collectionTranslationMap: Record<TCollectionName, string> = {
  'sentinel-1': `${BASE_KEY}.SENTINEL_1`,
  'sentinel-2-l1c': `${BASE_KEY}.SENTINEL_2`,
  'sentinel-2-l2a': `${BASE_KEY}.SENTINEL_2`,
  'sentinel-3': `${BASE_KEY}.SENTINEL_3`,
  'sentinel-5p': `${BASE_KEY}.SENTINEL_5P`,
  'esacci-globallc': `${BASE_KEY}.AUXILIARY.GLOBAL_LAND_COVER`,
  'clms-corinelc': `${BASE_KEY}.AUXILIARY.CORINE_LAND_COVER`,
  'clms-water-bodies': `${BASE_KEY}.AUXILIARY.WATER_BODIES`
};

const getCollectionTranslationKey = (collection: TCollectionName): string | null => {
  return collectionTranslationMap[collection] || null;
};
```

Comment (KatarzynaPaszko) [reply libs/map/data-access-map/src/lib/mutation/workflow.error.tsx:94]: Do similar with below cases as well

File: libs/shared/design-system/src/lib/snackbar/snackbar-provider.component.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      }}
+      anchorOrigin={{
+        vertical: 'bottom',
+        horizontal: 'center',
+      }}
+      action={(snackbarKey) => <SnackbarCloseButton snackbarKey={snackbarKey} />}
+    >
+      {children}
+    </NotistackProvider>
+  );
```
Comment (KatarzynaPaszko) [inline libs/shared/design-system/src/lib/snackbar/snackbar-provider.component.tsx:24]: you can use here our Notification component, so then you do not have to overwrite all styling in _libs/shared/design-system/src/lib/snackbar/snackbar.css_

**Review by KatarzynaPaszko — APPROVED** [review]

File: apps/eodh-fe/public/assets/i18n/en.json [hunk 3]
```diff
@@ -157,6 +121,12 @@
         }
       },
       "WORKFLOW": {
+        "ERROR": {
+          "AOI_TOO_BIG": "Area exceeds {{maxSize}} square miles.",
```
[the remark this answers is not in the document]
Comment (mlu-ssoft) [reply apps/eodh-fe/public/assets/i18n/en.json:125]: I would leave it as it is for now. If we will add support for it then we will add second key for translations. Right now there are two places which are displaying units. To do this properly we would have to add setting in the app to check which metric is chooses. Its not to big deal but right now we have a lot of work anyway, we can easilly add this in the next sprint

File: libs/shared/design-system/src/lib/snackbar/snackbar-provider.component.tsx (hunk 2 above)
Comment (mlu-ssoft) [reply libs/shared/design-system/src/lib/snackbar/snackbar-provider.component.tsx:24]: will it work with Notistack API? Notistack can display multiple actions.

**Pull request merged**